Client/Application/AppDelegate.swift

```diff
@@ -186,6 +188,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate, UIViewControllerRestorati
         self.profile = p
         return p
     }
+    
+    func updateShortcutItems(_ application: UIApplication) {
+        let newTabItem = UIMutableApplicationShortcutItem(type: "\(Bundle.main.bundleIdentifier ?? "").NewTab",
+            localizedTitle: NSLocalizedString("ShortcutItemTitleNewTab",
+                                              tableName: "InfoPlist",
+                                              bundle: Bundle.main,
+                                              comment: "New Tab"),
+            localizedSubtitle: nil,
+            icon: UIApplicationShortcutIcon(templateImageName: "quick_action_new_tab"),
+            userInfo: [:])
+        
+        let privateTabItem = UIMutableApplicationShortcutItem(type: "\(Bundle.main.bundleIdentifier ?? "").NewPrivateTab",
```

Comment:
Extract text values to `BraveStrings`, there's probably a text value for it already

Reply:
Fixed.